CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## v.0.21.2
 
+### Readme
+
+- Updated and edited various sections of readme, adding new tables and examples for clarity.
```

Comment:
another convention is to put a link to the PR where it was made
```suggestion
- Updated and edited various sections of readme, adding new tables and examples for clarity. [#6](https://github.com/firebolt-db/dbt-firebolt/pull/6)
```

Reply:
@marknoack I'm going to approve this change this time. thanks for your hard work!
<img width="865" alt="image" src="https://user-images.githubusercontent.com/8158673/139954508-a8f7d10d-5231-4684-8fd2-980e74f1aa7e.png">
<img width="611" alt="image" src="https://user-images.githubusercontent.com/8158673/139954531-c93fe039-bf20-4d5f-aebd-1af0baf8d187.png">